src/modules/users/customer.service.ts

```diff
@@ -0,0 +1,79 @@
+import { Inject, Injectable } from '@nestjs/common';
+import { User } from './entities/user.entity';
+import { UsersRepository } from './users.repository';
+import * as bcrypt from 'bcrypt';
+import { saltRounds } from './constants';
+import { CreateUserDto, UpdateUserDto } from './dto/user.dto';
+import {
+  IFilterParams,
+  ISortParams,
+} from './interfaces/user.search.sort.interface';
+import { ClientProxy } from '@nestjs/microservices';
+import { RoleEnum } from './enums/role.enum';
+import { IUserService } from './interfaces/user.service.interface';
+
+@Injectable()
+export class CustomersService implements IUserService {
+  constructor(
+    private readonly usersRepository: UsersRepository,
+    @Inject('SUBSCRIBERS_SERVICE') private readonly client: ClientProxy,
+  ) {}
+
+  async userLogin(email: string): Promise<User> {
+    const userDB = await this.usersRepository.findUserByEmail(email);
+
+    return userDB;
+  }
+
+  async readAllUsers(
+    filterParams: IFilterParams,
+    sortParams: ISortParams,
+    limit: number,
+    skip: number,
+  ): Promise<User[]> {
+    filterParams.role = RoleEnum.customer;
```

Comment:
this line makes this function unclean.

- [ ] Read about [Clean functions ](https://habr.com/ru/post/437512/#:~:text=%D0%A7%D0%B8%D1%81%D1%82%D1%8B%D0%B5%20%D1%84%D1%83%D0%BD%D0%BA%D1%86%D0%B8%D0%B8%20%E2%80%94%20%D1%81%D1%82%D1%80%D0%BE%D0%B8%D1%82%D0%B5%D0%BB%D1%8C%D0%BD%D1%8B%D0%B5%20%D0%B1%D0%BB%D0%BE%D0%BA%D0%B8%20%D0%B2,%D0%BE%D0%BF%D1%80%D0%B5%D0%B4%D0%B5%D0%BB%D0%B8%D1%82%D1%8C%20%D1%87%D0%B8%D1%81%D1%82%D0%B0%D1%8F%20%D1%84%D1%83%D0%BD%D0%BA%D1%86%D0%B8%D1%8F%20%D0%B8%D0%BB%D0%B8%20%D0%BD%D0%B5%D1%82.)